test/unit/specs/components/wallet/__snapshots__/PageWallet.spec.js.snap

```diff
@@ -192,103 +192,39 @@ exports[`PageWallet has the expected html structure 1`] = `
         <main
```

Comment:
I guess the test should also catch the loaded state

Reply:
We want this for the demo, so let's make an issue for it and merge this one :)